frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -468,7 +468,7 @@
   "fetch": [
     { "uri": "{{resource.assets.uris.bootstrap-zip}}", "cache": true },
     { "uri": "{{resource.assets.uris.jre-tar-gz}}", "cache": true },
-    { "uri": "{{resource.assets.uris.scheduler-zip}}", "cache": true },
+    { "uri": "{{resource.assets.uris.scheduler-zip}}", "cache": false },
```

Comment:
useful in debugging when scheduler-zip is the only thing that changes in stub definition.